docs/pages/router/advanced/nesting-navigators.mdx

```diff
@@ -31,7 +31,7 @@ import { Stack } from 'expo-router';
 export default Stack;
 ```
 
-Both **app/_layout.js** and **app/index.js** below are nested in the **\_layout.js** layout, so it will be rendered as a stack.
+Both **app/home/\_layout.js** and **app/index.js** below are nested in the **app/\_layout.js** layout, so it will be rendered as a stack.
```

Comment:
```suggestion
Both **app/home/_layout.js** and **app/index.js** below are nested in the **app/_layout.js** layout so that it will be rendered as a stack.
```